App/Dependencies/AppDependencies+DebugMenu.swift

```diff
@@ -110,6 +110,35 @@
         )
       ])
 
+      // analytics
+      items.append(contentsOf: [
+        .init(
+          title: "[Analytics] Trigger token generation",
+          dispatchable: Logic.Analytics.RefreshAnalyticsToken()
+        ),
+
+        .init(
+          title: "[Analytics] Trigger token validation",
+          dispatchable: Logic.Analytics.ValidateAnalyticsToken(token: self.getAppState().analytics.token!)
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/Dependencies/AppDependencies+DebugMenu.swift;
  Line: 122;
-->

- :warning: swiftlint (0.39.2): Force unwrapping should be avoided. (force_unwrapping)